test/e2e/singlecluster/pod_test.go

```diff
@@ -254,6 +255,105 @@ var _ = ginkgo.Describe("Pod groups", func() {
 			util.ExpectWorkloadToFinish(ctx, k8sClient, client.ObjectKey{Namespace: ns.Name, Name: "group"})
 		})
 
+		ginkgo.It("Deleted Pending Unschedulable Pod can be replaced in group", func() {
```

Comment:
```suggestion
		ginkgo.It("Unscheduled Pod which is deleted can be replaced in group", func() {
```
Making the pod Unschedulable is just a way to make the test reliable. What matters is that the pod wasn't scheduled.

Reply:
done